### PR TITLE
Jacob Elder - Tic-Tac-Toe

### PR DESCRIPTION
**Pull request by Jacob-Elder**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 4

Completeness Level (1-5):5

What did you think of this deliverable?:I had a lot of fun making this game!

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+	text-align:center;
+}
+
 h1 {
   color: purple;
 }
+
+.box {
+	height: 150px;
+	width:150px;
```
Comment (BertHaine) [inline css/style.css:14]: Small nitpick - make sure syntax is consistent throughout CSS (i.e. property:_space_value)

**Review by travismhayes — COMMENTED** [review]
nice job. I think you can assign the var computerTurn and var randomNumber globally so you don't have to assign them in the function itself, other then that good job.

File: js/app.js [hunk 2]
```diff
@@ -1 +1,97 @@
...
+*********************/
+var randomNumber;
+
+
+
+/********************************
+1) generates a random #
+2) appends an O if there isnt already an img
+***************************/
+function computerChoice() {
```
Comment (BertHaine) [inline js/app.js:251]: can remove additional spacing  on function code upon delivery.

**Review by BertHaine — COMMENTED** [review]
great job! other than a few minor syntax issues this is clear, concise and straightforward.

File: js/app.js [hunk 3]
```diff
@@ -31,6 +31,217 @@ random number used for the computer's choice
...
+*******************/
+var score = 0;
+
+/******************
+variable to display your score
+******************/
+var message = document.querySelector('.score');
+
+
+function checkForWinner() {
```
Comment (BertHaine) [inline js/app.js:60]: Can make this DRY by converting to a simple for loop - You can likely simplify this by assigning a boolean value for each clicked box, creating an array of correct scores and iterating through that with the for loop.

File: js/app.js [hunk 4]
```diff
@@ -1 +1,332 @@
 console.log('Hello frontend');
+
+/**********************
+A variable to access all the boxes ** returns an array **
+***********************/
+
+var boxes = document.getElementsByClassName('box');
+
+/*********************************************
+Variables to access each induvidual box
```
Comment (brandiw) [inline js/app.js:10]: Really nice, easy to read comments. It's "individual" though, haha.

File: js/app.js [hunk 5]
```diff
@@ -1 +1,332 @@
...
+
+		var img = document.createElement('img');
+
+		img.src = 'js/x.jpg';
+
+		this.appendChild(img);
+
+		this.data = "x";
+
+		//checkForWinner();
```
Comment (brandiw) [inline js/app.js:295]: Go ahead and remove extra lines and commented out code.

**Review by brandiw — COMMENTED** [review]
In general, you're doing really well with coding and the concepts from the class material. One thing that's sticking out to me is inconsistent formatting. Making it consistent is good for both readability and finding missing brackets/parenthesis/etc., and you will usually find it's part of your company's style guide.

**Pull request closed without merging**